x-pack/plugins/ml/common/openapi/bundled.json

```diff
@@ -0,0 +1,225 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Indicates a successful call",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "items": {
+                    "$ref": "#/components/schemas/syncResponse"
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/api/ml/saved_objects/sync": {
+      "get": {
+        "description": "Synchronizes Kibana saved objects for machine learning jobs and trained models. You must have `all` privileges for the **Machine Learning** feature in the **Analytics** section of the Kibana feature privileges. This API runs automatically when you start Kibana and periodically thereafter.\n",
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/simulateParam"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Indicates a successful call",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "items": {
+                    "$ref": "#/components/schemas/syncResponse"
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "components": {
+    "parameters": {
+      "spaceParam": {
+        "in": "path",
+        "name": "spaceId",
+        "description": "An identifier for the space.",
+        "required": true,
+        "schema": {
+          "type": "string"
+        }
+      },
+      "simulateParam": {
+        "in": "query",
+        "name": "simulate",
+        "description": "When true, simulates the synchronization by returning only the list actions that would be performed.",
+        "required": false,
+        "schema": {
+          "type": "boolean"
+        },
+        "example": "true"
+      }
+    },
+    "securitySchemes": {
+      "basicAuth": {
+        "type": "http",
+        "scheme": "basic"
+      }
+    },
+    "schemas": {
+      "syncResponse": {
+        "type": "object",
+        "properties": {
+          "datafeedsAdded": {
+            "type": "object",
+            "description": "If a saved object for an anomaly detection job is missing a datafeed identifier, it is added. This list contains the datafeed identifiers and indicates whether the synchronization was successful.",
+            "additionalProperties": {
+              "type": "object",
+              "properties": {
+                "success": {
+                  "type": "boolean"
+                }
+              }
+            }
+          },
+          "datafeedsRemoved": {
+            "type": "object",
+            "description": "If saved objects exist for datafeeds that no longer exist, they are deleted. This list contains the datafeed identifiers and indicates whether the synchronization was successful.",
```

Comment:
nit pick. this isn't technically correct, datafeeds don't have their own saved objects, they are referenced from job saved objects.
If a job saved object contains a datafeed id, but the actual datafeed doesn't exist, this list will contain the datafeed id.
but it might not be worth complicating the description with this.

Reply:
Thanks for the info!  I've updated the description in 9bb6640ad37c87e95a9a74c40fe4711c354a8695